kcptun.rb

```diff
@@ -0,0 +1,72 @@
+class Kcptun < Formula
+  desc "extremely simple & fast udp tunnel based on kcp protocol"
+  homepage "https://github.com/xtaci/kcptun"
+  url "https://github.com/xtaci/kcptun.git",
```

Comment:
Why does this need building from a git checkout rather than the archive tarball?